src/ErrorHandler.php

```diff
@@ -376,19 +382,17 @@ private function handleError(int $level, string $message, string $file, int $lin
         $this->invokeListeners($this->errorListeners, $errorAsException);
 
         if (null !== $this->previousErrorHandler) {
-            return false !== \call_user_func($this->previousErrorHandler, $level, $message, $file, $line, $errcontext);
+            return false !== ($this->previousErrorHandler)($level, $message, $file, $line, $errcontext);
         }
 
         return false;
     }
 
     /**
-     * Handles fatal errors by capturing them through the Raven client. This
-     * method is used as callback of a shutdown function.
-     *
-     * @param array|null $error The error details as returned by error_get_last()
+     * Handles fatal errors by capturing them through the client. This method
+     * is used as callback of a shutdown function.
      */
-    private function handleFatalError(array $error = null): void
+    private function handleFatalError(): void
```

Comment:
I think that this was a leftover of some old implementation of this method that was `public` to allow unit testing. In reality it's never called with the `$error` parameter filled, so it can be safely removed